aws_lambda_builders/workflows/nodejs_npm_esbuild/esbuild.py

```diff
@@ -154,15 +133,86 @@ def build_esbuild_args_from_config(self) -> "EsbuildCommandBuilder":
         """
         args = []
 
-        args.extend(self._get_boolean_args())
-        args.extend(self._get_single_value_args())
-        args.extend(self._get_multi_value_args())
+        for config_key, config_value in self._bundler_config.items():
+            if config_key in NON_CONFIGURABLE_VALUES:
+                continue
+            configuration_type_callback = self._get_config_type_callback(config_value)
+            args.extend(configuration_type_callback(config_key, config_value))
```

Comment:
Same as above, we might want to log these params so that it will be easy to track any issue in the future.